lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -13,11 +13,17 @@ interface UrlMatch {
 
 const urlRegex =
   /(?:https?:)?(?:\/\/)?(?:www\.)?(?<!api\.)(?:to)?github\.com\/[-_a-z0-9]+\/[-_a-z0-9]+\/(?:discussions|issues|pull)\/[0-9]+(?:#[-_a-z0-9]+)?/i; // TODO #12872 (?<!re) after text not matching
+const reduceUrlRegex =
+  /(?:https?:)?(?:\/\/)?(?:www\.)?github\.com\/(?<org>[a-zA-Z1-9\-_.]*)\/(?<repo>[a-zA-Z1-9\-_.]*)\/([a-zA-Z1-9\-_.]*)\/(?<number>[\d]+)/g;
```

Comment:
please align both to same allowed chars

Reply:
but if you mean `urlRegex` and `reduceUrlRegex` we have a PR for this change:
https://github.com/renovatebot/renovate/pull/17819/files